octoprint_Spoolman/SpoolmanPlugin.py

```diff
@@ -49,6 +56,10 @@ def getSpoolmanConnector(self):
         )
 
     def triggerPluginEvent(self, eventType, eventPayload = {}):
+        # Füge das "plugin_Spoolman_" Präfix hinzu, wenn es noch nicht vorhanden ist
+        if not eventType.startswith("plugin_"):
```

Comment:
This is not necessary, missing prefix is a sign of incorrect call arguments. I've added a comment in the place which is likely the cause of this change.